### PR TITLE
Personal Status, Contest Status

### PR DESCRIPTION
**Pull request by henryyang42**
Add Personal Status and Contest Status

### CODE REVIEW

**Comment by nickelcheng** [comment]
+1, @bruce3557 please check this.

File: status/templates/status/status.html [hunk 1]
```diff
@@ -22,81 +22,30 @@
...
-<script type="text/javascript">
-  $(function () {
-    $(document).tooltip({
-      track: true
-    });
-  })
-</script>
-
-<div class="jumbotron" style="width:95%;margin:auto;">
+<div class="container-fulid">
```
Comment (bruce3557) [inline status/templates/status/status.html]: typo?
should it be ```container-fluid```?

Comment (henryyang42) [inline status/templates/status/status.html]: Yes. Thanks!

**Comment by bruce3557** [comment]
+1

**Pull request merged**